engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/EngineErrorHandlingTest.java

```diff
@@ -170,10 +113,13 @@ public void shouldSkipFailingEvent() {
     assertThat(writtenEvent.getSourceEventPosition()).isEqualTo(secondEventPosition);
 
     // error response
-    verify(mockCommandResponseWriter).tryWriteResponse(anyInt(), anyLong());
+    final ArgumentCaptor<Long> requestIdCaptor = ArgumentCaptor.forClass(Long.class);
+    final ArgumentCaptor<Integer> requestStreamIdCaptor = ArgumentCaptor.forClass(Integer.class);
+    verify(mockCommandResponseWriter)
+        .tryWriteResponse(requestStreamIdCaptor.capture(), requestIdCaptor.capture());
```

Comment:
```suggestion
        .tryWriteResponse(eq(99), eq(255L));
```

:wrench: No wargument captors needed